drivers/crc/Kconfig.stm32

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Brill Power Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config CRC_STM32
+	bool "STM32 CRC driver"
+	default n
```

Enable this by default as there is a devicetree check dependency.

```suggestion
	default y
```